clogMonitor/src/components/LogEventsFilters.js

```diff
@@ -299,12 +344,16 @@ const LogEventsFilters = ({ dataSetHandler }) => {
                 }
 
                 <TimeRange 
-                    startTime={startTime} 
+                    startTime={startTime}
                     startChangeHandler={getDatetimeHandler(setStartTime)}
                     endTime={endTime}
                     endChangeHandler={getDatetimeHandler(setEndTime)}
+                    value1={startTimeDST}
+                    value2={endTimeDST}
```

Comment:
These should be dst1 and dst2, not value1 and value2. You should probably update the comments in the TimeRange component so that it's easier to find these kinds of errors